bundles/org.openhab.binding.lifx/README.md

```diff
@@ -101,6 +105,7 @@ MultiZone lights (*colormzlight*) have serveral channels (e.g. *colorzone0*, *te
 Changing the *color* and *temperature* channels will update the states of all zones.
 The *color* and *temperature* channels of MultiZone lights always return the same state as *colorzone0*, *temperaturezone0*.
 
+LIFX Tile (*tile*) supports special tile effects: morph and flame. These effects are predefined to their appearance using LIFX application. Each effect has a separate speed configurable.
```

Comment:
It's now a *tilelight*. Please start new sentences on new lines so it's easier to add comments in reviews.